js/script.js

```diff
@@ -69,76 +70,76 @@ const playGame = (playerSelection, computerSelection) => {
   }
   if(playerSelection.toLowerCase() === "paper" ){
       if(computerSelection.toLowerCase() === "rock"){
-          playerScore++;
+          scoreboard.playerScore++;
```

Comment:
Scores are defined within an object.

I would recommend renaming it from `scoreboard.playerScore` to `scoreboard.player`.
The name `score` is redundant and already inferred by the object name

```js
const scoreboard = {
  player: 0,
  computer: 0,
}
```